code/game/objects/items/gift.dm

```diff
@@ -8,6 +8,15 @@
  * Gifts
  */
 
+/var/static/blacklisted_items = typecacheof(list(
+	/obj/item/projectile,
```

Comment:
:-1:
you can pick up the projectiles and throw them at people to make them work

Reply:
...I did not know that was ever a thing. Don't worry, I'll change this.